db/seedData.js

```diff
@@ -1,32 +1,103 @@
 const {
-    client,
-    // declare your model imports here
-    // for example, User
-  } = require('./');
-  
-  async function buildTables() {
-    try {
-      client.connect();
-  
-      // drop tables in correct order
-  
-      // build tables in correct order
-    } catch (error) {
-      throw error;
-    }
+  client,
+  orders,
+  // declare your model imports here
+  // for example, User
+} = require("./");
+// const {createOrder} = require("./orders")
+
+// drop tables in correct order
+async function dropTables() {
+  try {
+    console.log("Dropping All Tables...");
+    client.query(`
+      DROP TABLE IF EXISTS products;
+      DROP TABLE IF EXISTS users;
+    `);
+
+    console.log("Finished dropping tables!");
+  } catch (error) {
+    console.error("Error while dropping tables!");
+
+    throw error;
   }
-  
-  async function populateInitialData() {
-    try {
-      // create useful starting data by leveraging your
-      // Model.method() adapters to seed your db, for example:
-      // const user1 = await User.createUser({ ...user info goes here... })
-    } catch (error) {
-      throw error;
-    }
+}
+
+// build tables in correct order
+async function buildTables() {
+  try {
+    console.log("Starting to build tables...");
+    await client.query(`
+      CREATE TABLE orders(
+        id SERIAL PRIMARY KEY,
+        "creatorId" INTEGER REFERENCES users(id),
+        name VARCHAR(255) UNIQUE NOT NULL,
```

Comment:
do our orders need a name? lmk what you think

also, I believe if we try to seed, since theres no users table yet, your table building will error out, since it can't find the reference for creatorId. In this case, your blocking issue is that you need the users table to be built to proceed.

Reply:
Copy that!

Customers are people too, Scott! Of course their name matters. lol xD